src/t8_forest/t8_forest_private.h

```diff
@@ -192,6 +192,19 @@ t8_forest_get_tree_element_array (t8_forest_t forest, t8_locidx_t ltreeid);
 t8_element_array_t *
 t8_forest_get_tree_element_array_mutable (const t8_forest_t forest, t8_locidx_t ltreeid);
 
+/** Search for a linear element id (at forest->maxlevel) in a sorted array of
```

Comment:
Is this docstring correct? [Here](https://github.com/DLR-AMR/t8code/blob/281a545221263879ade0c4286b6382d60e3f4a3e/src/t8_forest/t8_forest.cxx#L1990) we use the search explicitly with the current element level and not the maxlevel.

Reply:
Good catch, it feels like a bad naming choice.
I mark this as draft until resolved.

Reply:
This is not resolved yet @holke

Reply:
It is now updated